invenio_communities/views/communities.py

```diff
@@ -218,10 +219,13 @@ def members(community=None, pid_value=None, endpoint=None):
 @pass_community
 def invitations(community=None, pid_value=None):
     permissions = community.has_permissions_to(
-        ['update', 'read', 'search_requests', 'search_invites']
+        ['update', 'read', 'search_requests', 'search_invites', 'invite_owners']
     )
     if not permissions['can_search_invites']:
         raise PermissionDeniedError()
+    roles = deepcopy(current_app.config["COMMUNITIES_ROLES"])
+    if not permissions['can_invite_owners']:
+        del roles["owner"]
```

Comment:
could you give us more context about this change?

Reply:
Yes, in the [mockups](https://codimd.web.cern.ch/f3a7_DoqSfed6m_VMa649A#List-members-Invite-dialog) it says:
> Only an owner can invite other owners.

That's why we check and modify the roles we send to frontend.

Reply:
this change will not work in all the cases, especially because people can easily use ```{{config.COMMUNITIES_ROLES}}``` in jinja
I think COMMUNITIES_ROLES is more config and the owner role should rather come from the `current_user` roles. if `owner` is not present in these roles, then the "owner" radio button should be disabled/hidden. Removing it from the config is just a temporary workaround